cocos/video/video-player.ts

```diff
@@ -146,9 +146,9 @@ export class VideoPlayer extends Component {
 
     /**
      * @en
-     * The Video playback rate
+     * The Video playback rate. The value range is from (0.0 ~ 10.0).
```

Comment:
Range is [0,0 ~ 10.0]. `0.0` and `10.0` are included.

Reply:
Should check it.

Reply:
fixed.